### PR TITLE
fix last feedback

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: components/DesktopMenu/DesktopMenu.tsx [hunk 1]
```diff
@@ -30,9 +30,27 @@ const DesktopMenu = () => {
...
+        </button>
         <ToggleButton onClick={onToggle} value={isDarkMode} id="light_dark_switch" />
-        <div className="pl-2 text-[#9C9C9C]">dark</div>
+        <button
+          type="button"
+          className="px-2 text-[#9C9C9C] cursor-pointer text-[15px] font-quicksand uppercase"
+          onClick={() => {
+            setIsDarkMode(true)
+            if (themeMode === "light") onChangeThemeConfig()
+          }}
```
Comment (sweetmantech) [inline components/DesktopMenu/DesktopMenu.tsx]: you should ALWAYS toggle `onChangeThemeConfig`
```suggestion
          onClick={
          setIsDarkMode(!isDarkMode)
          onChangeThemeConfig()
          }
```

File: components/DesktopMenu/DesktopMenu.tsx [hunk 2]
```diff
@@ -30,9 +30,27 @@ const DesktopMenu = () => {
...
       </div>
       <div className="flex items-center font-quicksand font-bold">
-        <div className="pr-2 text-[#9C9C9C]">light</div>
+        <button
+          type="button"
+          className="px-2 text-[#9C9C9C] cursor-pointer text-[15px] font-quicksand uppercase"
+          onClick={() => {
+            setIsDarkMode(false)
+            if (themeMode === "dark") onChangeThemeConfig()
+          }}
```
Comment (sweetmantech) [inline components/DesktopMenu/DesktopMenu.tsx]: you should ALWAYS toggle `onChangeThemeConfig`
```suggestion
          onClick={
          setIsDarkMode(!isDarkMode)
          onChangeThemeConfig()
          }
```

File: components/Header/Header.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import Image from "next/image"
-import Link from "next/link"
 import { FC } from "react"
 import { useMediaQuery } from "usehooks-ts"
+import Link from "next/link"
```
Comment (sweetmantech) [inline components/Header/Header.tsx:4]: Is this `<Link />` ever used?

File: components/DesktopMenu/DesktopMenu.tsx [hunk 4]
```diff
@@ -25,14 +26,32 @@ const DesktopMenu = () => {
...
         <ToggleButton onClick={onToggle} value={isDarkMode} id="light_dark_switch" />
-        <div className="pl-2 text-[#9C9C9C]">dark</div>
+        <button
+          type="button"
+          className="px-2 text-[#9C9C9C] cursor-pointer text-[15px] font-quicksand uppercase"
+          onClick={() => {
+            setIsDarkMode(true)
+            onChangeThemeConfig("dark")
+          }}
+        >
```
Comment (sweetmantech) [inline components/DesktopMenu/DesktopMenu.tsx:52]: this code is repeated between both buttons right?
- please avoid defining functions in react code.
- please create a shared function used by both the `light` and `dark` button so you do not repeat this code.

let's focus on writing reusable components.

principle: [DRY - do not repeat yourself](https://docs.getdbt.com/terms/dry#:~:text=DRY%20code%20means%20you%20get,and%20energy%20on%20tedious%20syntax.)

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

**Pull request merged**